src/render/opengles2/SDL_render_gles2.c

```diff
@@ -663,26 +663,30 @@ static int
 GLES2_QueueDrawPoints(SDL_Renderer * renderer, SDL_RenderCommand *cmd, const SDL_FPoint * points, int count)
 {
     const SDL_bool colorswap = (renderer->target && (renderer->target->format == SDL_PIXELFORMAT_ARGB8888 || renderer->target->format == SDL_PIXELFORMAT_RGB888));
-    int color;
     const size_t vertlen = (2 * sizeof (float) + sizeof (int)) * count;
     GLfloat *verts = (GLfloat *) SDL_AllocateRenderVertices(renderer, vertlen, 0, &cmd->data.draw.first);
     int i;
+    SDL_Color color;
```

Comment:
not const?

Reply:
As per @sezero's other comment I believe it wouldn't be entirely c89-safe to initialize it in the same statement as declaring it since the variables it's initialized with aren't constant, but I can change it if you think it'd be better.

Reply:
Oh, right. I missed that we're not initializing it here.